index.js

```diff
@@ -617,6 +617,15 @@
     function(x) { return RegExp_._test(x) && !x.global; }
   );
 
+  //# NonNegativeInteger :: Type
+  //.
+  //. Type comprising `0`, `-0`, and every [`PositiveInteger`][] value.
```

Comment:
According to [`make lint`][1] there's a missing link definition. I suspect there's no link definition for ``[`PositiveInteger`]`` at the bottom of this file. Please add one. :)


[1]: https://circleci.com/gh/sanctuary-js/sanctuary-def/792